mockgen/mockgen.go

```diff
@@ -622,22 +628,23 @@ func (g *generator) Output() []byte {
 	return src
 }
 
-func lookupPackageName(importPath string) (string, bool) {
-	var pkg struct {
-		Name string
-	}
+func lookupPackagesName(importPaths []string) map[string]string {
+	names := make(map[string]string)
 	b := bytes.NewBuffer(nil)
-	cmd := exec.Command("go", "list", "-json", importPath)
+	args := []string{"list", "-f", "{{.Name}} {{.ImportPath}}"}
+	args = append(args, importPaths...)
+	cmd := exec.Command("go", args...)
 	cmd.Stdout = b
-	err := cmd.Run()
-	if err != nil {
-		return "", false
-	}
-	err = json.Unmarshal(b.Bytes(), &pkg)
-	if err != nil {
-		return "", false
+	cmd.Run()
+	sc := bufio.NewScanner(b)
+	for sc.Scan() {
+		pkg := strings.Split(sc.Text(), " ")
+		if len(pkg) != 2 {
```

Comment:
Don't want to `log.Fatal` as the program will crash and will not use the fallback mechanism in place.

Reply:
Ok, no problem